android_app/app/src/main/res/values/arrays.xml

```diff
@@ -118,4 +118,14 @@
         <item>@string/label_time_period_set_reference_day</item>
         <item>@string/label_time_period_set_custom_range</item>
     </string-array>
+
+    <string-array name="trendline_computation_methods_entries">
+        <item>@string/simple_moving_average</item>
+        <item>@string/exponentially_smoothed_moving_average</item>
+    </string-array>
+
+    <string-array name="trendline_computation_methods_values">
+        <item>Simple Moving Average</item>
```

Comment:
Ok I see what you did but maybe remove the spaces so its looks more like a variable